### ISSUE TITLE
Not sure how to create a PKPass ?

### ISSUE THREAD

**Issue by yozef** [appcelerator-archive/ti.passbook#6]
I see how to Add a Pass, but I'm not sure how to create the pkpass (signed)?

In the example I see them signed and created as pkpass, and simply reference them with:
~~~
var file = Ti.Filesystem.getFile(Ti.Filesystem.resourcesDirectory, "passes", "Lollipop.pkpass"); // how to sign the pass.json with the icon and logo?
try {
	Passbook.addPass({
		passData: file.blob
	});
} catch(err) {
	Log(err);
}
~~~
Does it require a server to sign it or is it possible to create the pkpass locally?
Basically the pass.json will be created dynamically with custom data.
